src/backend/app/Helpers/TokenGenerator.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Helpers;
+
+class TokenGenerator {
+    /**
+     * Generate a random 12-digit token.
+     *
+     * @return string
+     */
+    public static function generate() {
+        $token = '';
+        for ($i = 0; $i < 12; ++$i) {
+            $token .= random_int(0, 9);
+        }
+
+        return $token;
+    }
+}
```

Comment:
We'll leave this for now, but one comment for a later improvement: This should probably be a Factory https://laravel.com/docs/11.x/eloquent-factories